formats/html_publication/frontend/examples/prime_ministers_office.json

```diff
@@ -5,7 +5,9 @@
     "headings": "<ol></ol>",
     "public_timestamp": "2016-01-17T14:19:42.460Z",
-    "first_published_version": false
+    "first_published_version": false,
+    "isbn": "978-1-78386-459-1",
```

Comment:
@fofr The isbn field is not present in these examples, it's a field that's not normally filled in (but that people should be encouraged to use more often). And as for the print_meta_data_contact_address field, I've only just added the field so there's no way to reflect live data on it yet. 

So I have a chicken and egg problem here. 

Do you think I should just remove them from the examples?